runtime-builder/tests/integration/test.sh

```diff
@@ -16,7 +16,7 @@
 
 # test.sh deploys the test app and run the integration test on it.
 
-usage() { echo "Usage: $0 <project_id> [builder_image_tag | builder_image_url]"; exit 1; }
+usage() { echo "Usage: $0 <project_id> [builder_image_tag]"; exit 1; }
```

Comment:
builder_tag should be enough, so that it is consistent with other files.

Reply:
Done.